Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md

```diff
@@ -0,0 +1,55 @@
+
+While it's possible to have more than one Action Asset in a project, most projects only ever need a single Action Asset. This is because an Action Asset can contain multiple [Action Maps](ActionsEditor.html#configure-action-maps), which each containing a set of actions relevant to the various parts of your project (such as UI navigation, Gameplay, etc).
+
+The Input System's **project-wide actions** feature allows you to choose an individual Action Asset as being available project-wide, which means the actions within that asset are available more conveniently through the Input System API without needing to set up a reference to the Actions Asset.
+
+Therefore, unless you have specific project requirements that require more than one Action Asset, the recommended workflow is to use a single Action Asset assigned as the project-wide actions, as described below.
+
+## Create and Assign a Project-Wide Actions Asset
+
+To create and assign the current project-wide actions, go to **Edit** > **Project Settings** > **Input System Package**.
+
+If you don't yet have an Action Asset assigned as project-wide in your project, the Input System Package settings window displays an empty field for you to assign your action asset, and a button allowing you to create and assign one. 
+![image alt text](./Images/InputSettingsNoProjectWideAsset.png)</br>
+*The Input System Package Project Settings with no project-wide actions assigned*
+
+> **Note:** If you already have an Action Asset assigned, this button is not displayed, and instead the Actions Editor is displayed, allowing you to edit the project-wide actions.
+
+To create an Action Asset with default actions pre-configured, click  **"Create a new project-wide Action Asset"**. The asset is created in your project, and automatically assigned as the **project-wide actions**.
+
+The Action Asset appears in your Project view, and is named "InputSystem_Actions". This is where your new configuration of actions is saved, including any changes you make to it.
+
+![](images/InputSystemActionsAsset.png)<br/>
+*The new Actions Asset in your Project window*
+
+## Edit project-wide actions
+
+Once you have created and assigned project-wide actions, the Input System Package page in Project Settings displays the **Actions Editor** interface. Read more about how to use the [Actions Editor](ActionsEditor.md) to configure your actions.
+
+![image alt text](./Images/ProjectSettingsInputActionsSimpleShot.png)
+*The Input System Package Project Settings after creating and assigning the default actions*
+
+## The default input actions
+
+When you create and assign default project-wide actions using the method described above, the Action Asset comes pre-configured with some default Actions such as "Move", "Jump", and more, which suit many common app and game scenarios. They are configured to read input most types of input controller such as Keyboard, Mouse, Gamepad, Touchscreen and XR.
```

Comment:
Looks like some word got lost in the last sentence here?

Reply:
fixed in [ad3396b](https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/ad3396ba082647ee3cbfc46b84e96edacf424426)